build/rewrite.go

```diff
@@ -389,6 +427,18 @@ func sortStringLists(f *File) {
 				return
 			}
 			rule := callName(v)
+			var sortDenyList map[string]bool = tables.SortableDenylist
```

Comment:
This should be updated to not use tables. The only place in the file that should be referring to the tables package is the initialization of the Rewriter instance in Rewrite()

Reply:
Gotcha fixed